remove pinning, check win py35 again

kmuehlbauer opened the pull request.

kmuehlbauer committed:
remove pinning, check win py35 again

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (`recipe`) and found it was in an excellent condition.

kmuehlbauer commented:
Forgot to bumb build number. Let's wait for the CI's...

kmuehlbauer committed:
bumb build number

kmuehlbauer commented:
ping @jjhelmus 

The dependency-issues with hdf5/h5py etc. are corrected within conda-forge, so I removed all pinnings and added the py35 win again. It works! Just forgot to bump the build-number in the first attempt. Please have another look & merge when all is green.

ocefpaf commented:
@kmuehlbauer can you use this PR to remove the `krb5` too. That is fixed in our `gdal`.

kmuehlbauer committed:
Remove krb5 dependency
